ydb_sqlalchemy/sqlalchemy/__init__.py

```diff
@@ -341,13 +342,16 @@ def get_bind_types(
 
         return parameter_types
 
+    def visit_upsert(self, insert_stmt, visited_bindparam=None, **kw):
+        return self.visit_insert(insert_stmt, visited_bindparam, **kw).replace("INSERT", "UPSERT")
```

Comment:
**issue (blocking)** We should not replace only first occurrence, because of  the unexpected string values content, like:
```
UPSERT INTO TABLE my_table VALUES('INSERT is my favourite operation')
```

Also the test of this case should be